inference-engine/src/offline_transformations/include/moc_transformations.hpp

```diff
@@ -0,0 +1,35 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <memory>
+
+#include <transformations_visibility.hpp>
+
+#include <ngraph/pass/graph_rewrite.hpp>
+
+namespace ngraph {
+namespace pass {
+
+class TRANSFORMATIONS_API MOCTransformations;
```

Comment:
@GlebKazantaev if I want to put transformation that is required for TF only and intended to be used in MO (future) and in TF Bridge (now, in 2021.3). Is it a good place to put it in? Is this header and library itself is going to be available as a part of OV installed package?

Reply:
Yes, we can put such transformations to offline_transformations library but in case of TF bridge we must build it either with OV sources or developer package. But if you are asking about the final OV package I think we don't put headers even for our main transformation library so we won't put headers for offline_transformations library also. @ilya-lavrenov  correct me if I'm wrong.

Reply:
> Is this header and library itself is going to be available as a part of OV installed package?

IMO, current positioning - only library, not headers. 

> I think we don't put headers even for our main transformation library so we won't put headers for offline_transformations library also.

Right 

> TF Bridge

We need to understand how it is going to be built (at least I don't have such understanding)